src/connectors/range-slider/connectRangeSlider.js

```diff
@@ -186,28 +210,18 @@ export default function connectRangeSlider(renderFn) {
       },
 
       render({ results, helper, instantSearchInstance }) {
+        const bounds = { min: userMin, max: userMax };
         const facetsFromResults = results.disjunctiveFacets || [];
-        const facet = find(
-          facetsFromResults,
-          ({ name }) => name === attributeName
-        );
-        const stats =
-          facet !== undefined && facet.stats !== undefined
-            ? facet.stats
-            : {
-                min: null,
-                max: null,
-              };
-
-        if (userMin !== undefined) stats.min = userMin;
-        if (userMax !== undefined) stats.max = userMax;
+        const facet = find(facetsFromResults, _ => _.name === attributeName);
```

Comment:
`(_) => _.` is really not explicit, and you are using `lodash/find` you can refactor this in a more explicit way:

```js
const facet = find(facetsFromResults, { name: attributeName })
```

Reply:
👍